lib/senders/index.js

```diff
@@ -9,9 +9,8 @@
 // there's nothing left that imports mailer/config, it is safe to merge
 // legacy_index.js and this file into one.
 var createSenders = require('./legacy_index')
-var P = require('../promise')
 
-module.exports = function (log, config, error, db, translator, sender) {
+module.exports = function (log, config, error, bounces, translator, sender) {
```

Comment:
Can you also update anything from `scripts` that loads this module? I know that both the SMS ones do and I imagine `write-emails-to-disk` probably does and there maybe some others too. They're all pretty straightforward to run and verify that they're still working properly.